fuzzers/libfuzzer_libpng_centralized/src/lib.rs

```diff
@@ -201,7 +201,7 @@ pub extern "C" fn libafl_main() {
 
         // Create the executor for an in-process function with one observer for edge coverage and one for the execution time
         #[cfg(target_os = "linux")]
-        let mut executor = InProcessExecutor::batch_mode(
+        let mut executor = InProcessExecutor::with_batch_mode(
```

Comment:
Could be something like `InProcessExecutor::batched_timeouts` (but also not for this PR)

Reply:
done